### PR TITLE
Print sources details of snapshots and published repositories

### PR DESCRIPTION
**Pull request by sliverc**
## Description of the Change

For aptly wrapper tools to work such as [pyaptly](https://github.com/adfinis-sygroup/pyaptly) it needs to know sources of a snapshots and published repositories.
Such details are currently part of the description. However such description is hard to parse (or also to read for humans compare aptly publish list) and not always applicable (e.g. when a snapshot has been renamed, merged etc.).

Hence it would be good to have a Sources printed in the snapshot show command and have an additional command publish show to show details of a published repository as well.

Such details may also be very helpful for command line users to get a clue what has been done.

## Checklist

- [x] functional test added/updated (if change is functional)
- [x] man page updated (if applicable)
- [x] documentation updated
- [x] author name in `AUTHORS`

### CODE REVIEW

File: man/aptly.1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "APTLY" "1" "November 2016" "" ""
+.TH "APTLY" "1" "2017-01-10" "" ""
```
Comment (smira) [inline man/aptly.1]: I didn't finish putting `CONTRIBUTING` notes yet, but upstream `ronn` has one bug (https://github.com/rtomayko/ronn/pull/83) which is not patched. It leads to broken rendering of man page (and it's the reason for huge diff here).

I've got patched version here: https://github.com/smira/ronn

Probably I need to put some Makefile magic to pull in patched ronn and do transformation.

**Comment by smira** [comment]
@sliverc on docs: https://github.com/aptly-dev/aptly-dev.github.io

**Comment by sliverc** [comment]
@smira 
I recreated the man page with the patched version of ronn now and also created a PR for documentation changes at https://github.com/aptly-dev/aptly-dev.github.io/pull/29

**Comment by smira** [comment]
Pushed as nightly `aptly_0.9.8~dev+51+g207ebff`
